src/main/java/org/jabref/logic/integrity/TitleChecker.java

```diff
@@ -12,6 +12,7 @@
 public class TitleChecker implements ValueChecker {
 
     private static final Pattern INSIDE_CURLY_BRAKETS = Pattern.compile("\\{[^}\\{]*\\}");
+    private static final String DELIMITERS = "(\\.|\\!|\\?|\\;|\\:)";
```

Comment:
as this is a regex, too, you should create a Pattern analogous to the other ones and then call the split method on the pattern. The Pattern.compile is more efficient when the regex is used multiple times

Reply:
Thanks for pointing out this. Good to learn! I changed it.